<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{
	border:none;
	background-color:transparent;
	color:#fff;
}
#centralwidget{
	background-color:#040f13;
}
#side_menu{
	background-color:#071e26;
	border-radius:20px;
}


QPushButton{
	padding:10px;
	background-color:#040f13;
	border-radius:5px;
}

QComboBox{
	padding:10px;
	background-color:#040f13;
	border-radius:5px;
}

QPushButton:hover{
	padding:10px;
	background-color:#12394a;
	border-radius:5px;
}
QComboBox:hover{
	padding:10px;
	background-color:#12394a;
	border-radius:5px;
}

#main_body{
	background-color:#071e26;
	border-radius:10px;
}

#texto{
	background-color:#071e26;
	border-radius:10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="header">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_menu" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="menu">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>MENU</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/Imagen1.png</normaloff>icons/Imagen1.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mapa">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>17</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>INTERFAZ  RISARALDA</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="3">
        <widget class="QFrame" name="side_menu">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QWidget" name="boxmenu" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="tipo">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Tipo de Algoritmo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="algoritmo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>10</pointsize>
                 <stylestrategy>PreferAntialias</stylestrategy>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::WheelFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="currentText">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cityo">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Ciudad de Origen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="origen">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cityd">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Ciudad de Destino</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="destino">
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="graficar">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Graficar</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/Imagen4.png</normaloff>icons/Imagen4.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="limpiar">
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Limpiar</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/Imagen3.png</normaloff>icons/Imagen3.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="salir">
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Salir</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/Imagen2.png</normaloff>icons/Imagen2.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="adyacencia">
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Tabla de Adyacencia</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="encabezado">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>625</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Selecciona en el menu el tipo de algortimo de búsqueda no informada, ciudad de origen</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="encabezado_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>625</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>11</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>y ciudad de destino para encontrar la ruta más corta</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="main_body" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>625</width>
           <height>425</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1250</width>
           <height>800</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="image">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>image/risaralda.jpeg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFrame" name="texto">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>625</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1250</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="etiqueta">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Rendimiento y Consumo:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="etiqueta4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Resultados:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="etiqueta3">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid white;
border-radius: 5px</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="etiqueta2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid white;
border-radius: 5px</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>limpiar</sender>
   <signal>clicked()</signal>
   <receiver>etiqueta2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>751</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
